### ISSUE TITLE
CIMHub ignores the neutral conductor for Overhead Line

### ISSUE THREAD

**Issue by shpoudel** [GRIDAPPSD/GOSS-GridAPPS-D#1561]
I observed the following object definitions for an overhead line. The line impedances will be different when N is missing in the **phases** property. I think this is the main reason for the mismatch between GLD and OpenDSS power flow solution for the 9500 node model. 

![image](https://user-images.githubusercontent.com/35777492/135673881-15b32dec-d45b-4ef8-869d-0f1e876f0541.png)

Following are the impedances calculated by GLD with and without N in the **phases**. 

![image](https://user-images.githubusercontent.com/35777492/135674306-608933a7-65cc-47f2-9bb2-abeef35ba23a.png)

**Comment by temcdrm** [comment]
Ok.  I'm tempted to argue this is a bug in GridLAB-D, but it can be fixed in CIMHub.

**Comment by shpoudel** [comment]
Thanks, Tom. 
Are you saying if the neutral conductor is defined in line_configuration, then GridLAB-D should be smart enough to consider a **N** phase in the line object?

**Comment by temcdrm** [comment]
Yes, because the neutral is assumed to be grounded.  See, the N phase doesn't actually appear in the line object, as evidenced from the 3x3 matrix in both of your examples.

It's possible to have ungrounded neutrals, or to explicitly model the neutral pole grounds at specific poles, but so far as I know, GridLAB-D doesn't offer that option.

**Comment by shpoudel** [comment]
The b_matrix that I have printed in both cases is a kron reduced one. 

Yes, without N, the primitive impedance is made equal to the phase impedance matrix as there is no need for Kron reduction. This is how GLD is handling this case.
